pages/common/aws-kinesis.md

```diff
@@ -0,0 +1,28 @@
+# aws kinesis
+
+> Official AWS CLI for Amazon Kinesis streaming data services.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/kinesis/index.html#cli-aws-kinesis>.
+
+- Show all streams in the account:
+
+`aws kinesis list-streams`
+
+- Write one record to a Kinesis stream:
+
+`aws kinesis put-record --stream-name {{streamname}} --partition-key {{partition_key}} --data {{base64_encoded_message}}`
```

Comment:
You can use just `name` because the argument has `-name` is easy to read:
```suggestion
`aws kinesis put-record --stream-name {{name}} --partition-key {{partition_key}} --data {{base64_encoded_message}}`
```
Or
```suggestion
`aws kinesis put-record --stream-name {{stream_name}} --partition-key {{partition_key}} --data {{base64_encoded_message}}`
```
The same for the other examples with `--stream-name`

Reply:
Good call, updated in 669be42a. I applied the same to the other parameters, like changing `--partition-key {{partition_key}}` to `--partition-key {{key}}`. I can change those back if you prefer.